### PR TITLE
[ja] update some translations.

### PR DESCRIPTION
**Pull request by okinaka**

### CODE REVIEW

File: ja/controllers/components/security.rst [hunk 1]
```diff
@@ -156,7 +156,7 @@ Security コンポーネントは、一般的にコントローラの ``beforeFi
 
         public function beforeFilter(Event $event)
         {
-            if (isset($this->request->params['admin'])) {
+            if ($this->request->getParams('admin')) {
```
Comment (chinpei215) [inline ja/controllers/components/security.rst]: getParam

File: ja/controllers/components/security.rst [hunk 2]
```diff
@@ -180,14 +180,14 @@ Security コンポーネントは、一般的にコントローラの ``beforeFi
 
         public function beforeFilter(Event $event)
         {
-            if (isset($this->params['admin'])) {
+            if ($this->getParams('admin')) {
```
Comment (chinpei215) [inline ja/controllers/components/security.rst]: Same as above.

**Comment by okinaka** [comment]
@chinpei215 Thanks! I updated.

**Comment by chinpei215** [comment]
@okinaka Thanks.

**Pull request merged**